.github/workflows/tox-test.yml

```diff
@@ -3,20 +3,20 @@ name: Tox tests
 on: [push, pull_request]
 
 jobs:
-  py27:
-    runs-on: ubuntu-20.04
+  py36:
```

Comment:
I don't think you need to add this because the 'coverage' job already covers python 3.